report/4.Solution/GeneratedCppCode.tex

```diff
@@ -0,0 +1,86 @@
+% !TeX root = ../main.tex
+
+\subsection{Generated C++ code}
+While the previous section is primarily focused on the code generation from an abstract point of view, this section focuses on the generated C++ code.
+Since C++ is the output language of the HCL compiler, all the language concepts of HCL have to be represented in valid C++ code.
+
+\textbf{Primitive types}
+The primitive types of HCL includes numbers, text and booleans. 
+Booleans are straightforward, as HCL's boolean behaves exactly like the C++ boolean type.
+This means that the regular C++ boolean type can be used as the output type of HCL booleans.
+
+As HCL aims for simplicity, only one number type is introduced. 
+This type needs to be able to store both integers and non-integers. 
+The simple solution is to turn all HCL numbers into C++'s double type.
+The solution with doubles is not ideal, but it will fulfill the requirements of the number type in HCL.
+
+C++ does have an implementation of a string type in its standard library.
+However, the Arduino does not include this implementation.
+The Arduino uses its own string implementation, which varies from the standard implementation.
```

Comment:
Remove "That varies from the standard implementation" and add a "." after "implementation".
It is quite obvious that the custom implementation would differ from the standard implementation.

Reply:
Hvad med at flette linje 18 og 19 sammen til "However, the Arduino does not include this implementation, but rather uses its own."

Reply:
That works even better :)